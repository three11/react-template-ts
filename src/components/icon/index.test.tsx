import * as React from 'react';
import { render } from '@testing-library/react';

import { Icon } from '.';

test('Icon component should render successfully', () => {
	const { asFragment } = render(<Icon src="@assets/logo.svg" />);

	expect(asFragment()).toMatchSnapshot();
});
