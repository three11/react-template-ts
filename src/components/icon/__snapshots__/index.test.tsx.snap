// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon component should render successfully 1`] = `<DocumentFragment />`;
